cinemy/cinema/urls.py

```diff
@@ -8,6 +8,16 @@
     reservations_per_user,
     confirm_reservations,
 )
+from rest_framework.routers import DefaultRouter
+from cinema.viewset import MovieCurrentlyPlayingViewSet
+
+router = DefaultRouter()
+router.register(
+    "movies_playing_this_week",
```

Comment:
It is recommended to use `-` instead of any other word separator in the urls. I see only now that you used `_` in urls for other endpoints. Take a look at this https://webmasters.stackexchange.com/questions/374/urls-should-i-use-hyphens-underscores-or-plus-symbols